### ISSUE TITLE
AttributeError: 'ProtobufEntry' object has no attribute 'text_payload'

### ISSUE THREAD

**Issue by waprin** [GoogleCloudPlatform/gcloud-python#1807]
Reproducing this is blocked by #1806, but I wanted to make sure it gets recorded this popped up. Still on `client.list_entries`:

```python
  File "list_logs.py", line 45, in list_logs
    format((timestamp, entry.text_payload, entry.struct_payload)))
AttributeError: 'ProtobufEntry' object has no attribute 'text_payload'
```

**Comment by waprin** [comment]
Actually if I remove the #1806 validation I reliably reproduce this.

**Comment by waprin** [comment]
I am now realizing that this is in my own code, however that was copied from [the docs](https://googlecloudplatform.github.io/gcloud-python/stable/logging-usage.html) so maybe just a docs error.

**Comment by waprin** [comment]
@dhermes Still need to clean up the docs. Is the idea that now you will just reference `payload` directly for each LogEntry type? Also, the docs say its logger.delete_entries() when now its logger.delete(). Can make the PR if you'd like.

**Comment by tseaver** [comment]
@waprin, can you show the example which is triggering the traceback?

> Is the idea that now you will just reference payload directly for each LogEntry type?

Yes, each entry type (`TextEntry`, `StructEntry`, and `ProtobufEntry`) will have a `payload` attribute.  For `ProtobufEntry` objects, you can use the `parse_mesage` method to get an actual protobuf:  by default, it comes across the wire as a mapping.

> Also, the docs say its `logger.delete_entries()` when now its `logger.delete()`. Can make the PR if you'd like.

Thanks, that would be great.

**Comment by waprin** [comment]
http://googlecloudplatform.github.io/gcloud-python/stable/logging-usage.html .

```
from gcloud import logging
>>> client = logging.Client()
>>> entries, token = client.list_entries()  # API call
>>> for entry in entries:
...    timestamp = entry.timestamp.isoformat()
...    print('%sZ: %s | %s' %
...          (timestamp, entry.text_payload, entry.struct_payload))
```

There are some other changes as well, I am writing the samples for python-docs-samples then I'll swing by and update it all.

**Comment by dhermes** [comment]
Thanks!

**Comment by waprin** [comment]
Still planning on fixing these docs, just waiting for your next release so I have all the working samples then I'll update the docs here.

**Comment by dhermes** [comment]
I can cut a release tomorrow. @tseaver Any reason not to?

**Comment by tseaver** [comment]
@dhermes nope.

**Comment by dhermes** [comment]
OK starting process now. I'll cc you on the release PR @waprin

**Comment by tseaver** [comment]
Docs fixed in #1848
